dotnet/src/SemanticKernel.Abstractions/AI/FunctionChoiceBehaviors/FunctionChoiceBehaviorConfiguration.cs

```diff
@@ -18,4 +20,14 @@ public sealed class FunctionChoiceBehaviorConfiguration
     /// The functions available for AI model.
     /// </summary>
     public IReadOnlyList<KernelFunction>? Functions { get; internal init; }
+
+    /// <summary>
+    /// Indicates whether the functions should be automatically invoked by the AI connector.
+    /// </summary>
+    public bool AutoInvoke { get; internal init; } = true;
+
+    /// <summary>
+    /// Specifies whether validation against a specified list of functions is required before allowing AI connector to invoke it.
```

Comment:
Maybe more information would be useful here, e.g. how to specify the list of functions the check against, or how to provide some code to do the validation.

Reply:
Also, AllowAny seems very broad and I can imagine many meanings.  Reading the comment, I'm wondering if something like RequireKernelFunctionValidation would be clearer?

Reply:
One final thought.  Is this the connector doing the validation?  Wouldn't this type of behavior make sense in a filter?

Reply:
It seems I have a solution to get rid of the property completely. I'll push it in a separate PR.

Reply:
The property is not necessary and was removed.